Sources/HTML/HTML.swift

```diff
@@ -30,6 +30,7 @@ public struct Attribute {
 }
 
 public protocol Value {
+  func renderedValue() -> EncodedString?
```

Comment:
so I had to add this so that I could better customize how certain attributes were rendered. it's a bit hacky, but I think we could clean it up easily later.